Packs/GreyNoise/Integrations/GreyNoise/GreyNoise.py

```diff
@@ -611,56 +908,70 @@ def main() -> None:
     :return:
     :rtype:
     """
+    # get pack version
+    response = demisto.internalHttpRequest("GET", "/contentpacks/metadata/installed")
+    packs = json.loads(response["body"])
 
-    api_key = demisto.params().get('apikey')
-    proxy = demisto.params().get('proxy', False)
+    pack_version = "0.0.0"
+    for pack in packs:
+        if pack["name"] == "GreyNoise":
+            pack_version = pack["currentVersion"]
 
-    demisto.debug(f'Command being called is {demisto.command()}')
+    api_key = demisto.params().get("apikey")
+    proxy = demisto.params().get("proxy", False)
+
+    demisto.debug(f"Command being called is {demisto.command()}")
     try:
 
         client = Client(
             api_key=api_key,
             api_server=API_SERVER,
             timeout=TIMEOUT,
-            proxy=handle_proxy('proxy', proxy).get('https', ''),
+            proxy=handle_proxy("proxy", proxy).get("https", ""),
             use_cache=False,
-            integration_name=f"xsoar-integration-v{demisto.demistoVersion()['version']}"
+            integration_name=f"xsoar-integration-v{pack_version}",
```

Comment:
This does not seem like the correct integration name to me, am I missing something?